src/screens/OMPSwitch/MerchantSwitch.res

```diff
@@ -30,8 +30,18 @@ module NewAccountCreationModal = {
     let merchantName = FormRenderer.makeFieldInfo(
       ~label="Merchant Name",
       ~name="company_name",
-      ~placeholder="Eg: My New Merchant",
-      ~customInput=InputFields.textInput(),
+      ~customInput=(~input, ~placeholder as _) =>
+        InputFields.textInput()(
+          ~input={
+            ...input,
+            onChange: event =>
+              ReactEvent.Form.target(event)["value"]
+              ->String.trimStart
```

Comment:
This will just trim the empty spaces from the beginning but what if we add extra spaces in the end

Reply:
I have done this intentionally because what is user wants to enter a name like "Merchant XYZ" we should allow spaces at the end for that.

Reply:
Trim will just remove spaces from the end of the string not in the middle

Reply:
yes, trim start will only remove front spaces. i have used it on onChange and it will not allow space between words if trim is used here. for the spaces at the end, i will make changes in the api payload itself. This onChange trimStart is required so that user will have better UI experience and will not be able to enter front spaces itself in the input box.